packages/core/src/utils/cloudflare/index.ts

```diff
@@ -29,7 +29,7 @@ export const createCustomHostname = async (auth: HostnameProviderData, hostname:
       },
       json: {
         hostname,
-        ssl: { method: 'txt', type: 'dv', settings: { min_tls_version: '1.0' } },
+        ssl: { method: 'http', type: 'dv', settings: { min_tls_version: '1.0' } },
```

Comment:
see slack discussion

```suggestion
        ssl: { method: 'http', type: 'dv', settings: { min_tls_version: '1.2' } },
```

Reply:
updated